tss-esapi/src/context/general_esys_tr.rs

```diff
@@ -85,6 +85,26 @@ impl Context {
         Ok(())
     }
 
+    #[cfg(has_esys_tr_get_tpm_handle)]
+    /// Retrieve the `TpmHandle` stored in the given object.
+    pub fn tr_get_tpm_handle(&mut self, object_handle: ObjectHandle) -> Result<TpmHandle> {
+        use crate::tss2_esys::Esys_TR_GetTpmHandle;
+        use tss_esapi_sys::TPM2_HANDLE;
+        let mut tpm_handle: TPM2_HANDLE = Default::default();
+        ReturnCode::ensure_success(
+            unsafe {
+                Esys_TR_GetTpmHandle(
+                    self.mut_context(),
+                    object_handle.into(),
+                    &mut tpm_handle)
+            },
+            |ret| {
+                error!("Error when getting TPM handle from ESYS handle: {}", ret);
+            },
+        )?;
+        Ok(TpmHandle::try_from(tpm_handle)?)
```

Comment:
You do not need to do that.

Just let it return what ever comes out of the ```try_from```.

Reply:
Derp, that's a function and its inverse. I'm so used to converting errors in my projects. Fixed in 2e87dcb.